src/csvcubed/models/rdf/prov.py

```diff
@@ -44,6 +47,13 @@ class Entity(NewResource):
         Resource[Activity],
         Triple(PROV.wasGeneratedBy, PropertyStatus.optional, map_resource_to_uri),
     ]
+    title: Annotated[
+        str, Triple(DCTERMS.title, PropertyStatus.recommended, map_str_to_en_literal)
+    ]
+    has_primary_source: Annotated[
```

Comment:
Why did this show up in here? Should this not be in models?

Reply:
`prov.py` was already here (with the `Activity` class definition). Rob did tell me why some of the models are here instead of in csvcubed-models, but I can't remember exactly what he said - something about it being more flexible?